web/src/ApiButton.test.tsx

```diff
@@ -142,6 +149,43 @@ describe("ApiButton", () => {
     expect(actualStatus).toEqual(expectedStatus)
   })
 
+  it("reads options from local storage", async () => {
```

Comment:
do these tests need to be `async`? i'm curious because i didn't need to use it when i was writing tests with local storage, but i may be missing something!

Reply:
Nope, I just copied them from a test that happened to need to be async. Thanks!